site/sidebar.js

```diff
@@ -65,6 +65,7 @@ module.exports = {
             },
           ],
         },
+        // "numbers/numbersApiV2",
```

Comment:
what is this for?

Reply:
we spoke in person and this will be uncommented when we're ready for customers to see it